game/view/transmission.lua

```diff
@@ -29,6 +32,16 @@ function Transmission:init(origin, target, color, duration)
   self.tar_ox = RANDOM.safeGenerate(-_MAX_OFFSET, _MAX_OFFSET)
   self.tar_oy = RANDOM.safeGenerate(-_MAX_OFFSET, _MAX_OFFSET)
 
+  --Create bendings
+  self.bending_number = bending_number or 6
+  self.bending_offsets = {}
+  for i = 1, self.bending_number do
+    self.bending_offsets[i] = RANDOM.safeGenerate(_MIN_BEND_OFFSET, _MAX_BEND_OFFSET)
+  end
+  self.bending_angles = {}
+  for i = 1, self.bending_number do
+    self.bending_angles[i] = RANDOM.safeGenerate(_BEND_ANGLE, _BEND_ANGLE)
```

Comment:
why use a random for an empty interval?

Reply:
That was a mistake >< It was supposed to be -_BEND_ANGLE to _BEND_ANGLE